src/style.css

```diff
@@ -0,0 +1,28 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
Don't reset your styles under * selector, it's a bad practice, more so, you only need to reset margin for body in this task